src/redesign/data/utils/dataUtils.js

```diff
@@ -42,12 +42,20 @@ export const getTpaHint = () => {
 };
 
 export const updatePathWithQueryParams = (path) => {
-  const queryParams = window.location.search;
+  let queryParams = window.location.search;
 
   if (!queryParams) {
     return path;
   }
 
+  if (queryParams === '?track=pwreset') {
+    queryParams = '';
+  }
+
+  if (queryParams.indexOf('track=pwreset') > -1) {
+    queryParams = queryParams.replace('?track=pwreset', '?').replace('&track=pwreset', '').replace('?&', '?');
```

Comment:
we can remove the above `if condition` by modifying this one like:

`queryParams.replace('?track=pwreset&', '?').replace('?track=pwreset', '').replace('&track=pwreset', '').replace('?&', '?')`